doc/ref_model/chapters/appendix-a.md

```diff
@@ -1,14 +1,15 @@
 [<< Back](../../ref_model)
-# Appendix A - Guidelines For VNF Vendors
+# Appendix A - VNF Design Guidelines
```

Comment:
```suggestion
# Appendix A - Guidelines For VNF Vendors	
```
The title was changed from "VNF Design Guidelines" to "Guidelines for VNF Vendors" back in 2019.  This title seems more appropriate, as the guidelines here expand a bit more beyond just design guidelines.  For instance, the VNF evolution phases provides some background on CNTT design principles.

Reply:
Agree. The very original idea was that all guidance to VNF vendors goes into that one place, and text on impact between VNF design and platform is great fit there.

Reply:
@kedmison References to the Appendix in RM as well as the Title of the App use "VNF Design Guidelines"

Changing Appendix Title (within doc and TOC) to "Guidelines for VNF Vendors" but please ensure changes are made in all references to this Appendix.

Reply:
Thanks for flagging this.  PR #1293 is created.